### DISCUSSION TITLE
[Discussion] Some interesting A/B flags you may want to try

### DISCUSSION THREAD

**Discussion by qnblackcat** [PoomSmart/YTABConfig#14] [Ideas]
| Flag | Tested in  | Value | Demo |
|:---|:---:|:---:|:-----|
|`enableIosRoundedSearchBar` | v17.41.2 | 1 | - Rounded the search bar? <br> - [Demo](https://user-images.githubusercontent.com/52943116/197344506-8fbd81fd-1556-4f8e-92d0-846cdbf26fe7.jpeg) |
|`uiSystemsClientGlobalConfigEnableModernButtonsForNative`<br>`uiSystemsClientGlobalConfigEnableRoundedDialogForNative` | v17.41.2 | 1 | - Rounded some app's buttons. - Demo: [1](https://user-images.githubusercontent.com/52943116/197346640-fd1d9e5c-9626-4fa8-b92e-5afa948305cc.jpeg)/ [2](https://user-images.githubusercontent.com/52943116/197346641-81bffb5c-035f-402f-8b9e-d83dfc152187.jpeg) |
|`uiSystemsClientGlobalConfigEnableRoundedThumbnailsForNative` | v17.41.2 | 1 | - Rounded the video's thumbnails in several sections (History, Playlist...). <br> - [Demo](https://user-images.githubusercontent.com/52943116/197346887-dea23345-c4a3-42e7-99d6-18eae402e2fd.jpeg) |
|`enableResumeToShorts` | v17.41.2 | 0 | - If you close YouTube while watching Shorts, Shorts won‘t automaticlly play next time you launch YouTube |
|`enableSwipeToRemoveInPlaylistWatchEp` | v17.41.2 | 1 | - Swipe right to remove video in Playlist. <br> - [Demo](https://user-images.githubusercontent.com/52943116/197347541-1d350424-6aad-4533-be70-f98a39e7d8ad.jpeg) |
|`enableCinematicContainer` <br> `enableCinematicContainerOnClient` | v17.41.2 | 1 | - Enable cinematic mode? <br> - [More details](https://github.com/qnblackcat/uYouPlus/issues/560). |
|`isLandscapeEngagementPanelEnabled` | v17.41.2 | 0 | - Disable the right panel in full screen mode. <br> - There is a tweak based on this flag IIRC.
|`isLandscapeEngagementPanelSwipeRightToDismissEnabled` | v17.41.2 | 1 | - Swipe right to dismiss the right panel. <br> - Enabled by default in uYou+/Cercube+.
|`replaceNextPaddleWithFastForwardButtonForSingletonVods`<br> `replacePreviousPaddleWithRewindButtonForSingletonVods` | v17.41.2 | 1 | - Replace the Next & Previous button with the Fast forward & Rewind button. <br> - Available in uYou+/Cercube+ settings.<br>- Will not work in Playlist. <br> - [Demo](https://user-images.githubusercontent.com/52943116/197346013-db47984e-b20c-4bfe-854f-6b34b7a598c7.jpeg) |
|`removeNextPaddleForSingletonVideos`<br> `removePreviousPaddleForSingletonVideos` | v17.41.2 | 1 | - Hide the Next & Previous button. <br> - Available in uYou+/Cercube+ settings.<br>- Will not work in Playlist <br> - [Demo](https://user-images.githubusercontent.com/52943116/197346169-6802088b-26fe-4012-9c71-c2c267d8747a.jpeg) |

***
### Notes:
- There are other flags but they're pretty useless. If you found something usefull, don't hesitate to share it. 
- `Value = 0` is disabled, `1` is enabled.
- Flag can be removed/added in each update of YouTube. Check [this thread](https://github.com/PoomSmart/YTABConfig/discussions/2) where I upload all the flags of each version.

**Comment by moh-ter** [comment]
Amazing effort. Thanks for this informative list. Hopefully it’ll grow as the community test more settings.

**Comment by vikas1255** [comment]
Is there a flag for playback speed control overlay also? Unable to find it.
